LabExT/ExperimentManager.py

```diff
@@ -69,6 +76,7 @@ def __init__(self, root, log_file_path, chip=None, skip_setup=False):
         self.mover = MoverNew(experiment_manager=self, chip=chip)
         self.peak_searcher = PeakSearcher(
             None, self, mover=self.mover, parent=self.root)
+        self.live_viewer_model: LiveViewerModel = None
```

Comment:
I added some type hinting here. I'm sorry, it's like an addiction for me :sweat_smile: